packages/playwright/src/reporters/base.ts

```diff
@@ -373,6 +373,12 @@ export function formatFailure(config: FullConfig, test: TestCase, options: {inde
   };
 }
 
+function quoteIfContainsWhitespace(path: string): string {
+  if (/\s/.test(path))
```

Comment:
maybe test for the opposite? I feel like brackets would hurt us as well.

Reply:
Done. In that case we may also need to escape some characters. Let's wait if it bothers anyone.